lib/cli/interactive-setup/aws-credentials.js

```diff
@@ -327,7 +351,10 @@ module.exports = {
       }
       const hasDefaultProvider = providers.some((provider) => provider.isDefault);
 
-      if (hasDefaultProvider) return false;
+      if (hasDefaultProvider) {
+        context.reason = 'hasDefaultProvider';
```

Comment:
As agreed, let's rename it

Reply:
I think you had some old version opened as it was changed with last push